### PR TITLE
add proto-tools

### PR DESCRIPTION
**Pull request by nevillelyh**

### CODE REVIEW

File: proto-tools/src/main/resources/META-INF/NOTICE [hunk 1]
```diff
@@ -0,0 +1 @@
+GCS compatible Apache Avro Tools
```
Comment (ravwojdyla) [inline proto-tools/src/main/resources/META-INF/NOTICE]: Is this correct?

**Comment by ravwojdyla** [comment]
Do you need tests?

**Comment by ravwojdyla** [comment]
👍

File: proto-tools/src/main/resources/META-INF/NOTICE (hunk 1 above)
Comment (nevillelyh) [inline proto-tools/src/main/resources/META-INF/NOTICE]: Duh

**Comment by nevillelyh** [comment]
Updated `.travis` and `README.md`. Gonna skip tests for now.

**Pull request merged**